Structure_Engine/Query/ReinforcingBarCount.cs

```diff
@@ -45,12 +45,12 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Return the number of rebars in one particular Reinforcement object.")]
-        [Input("reinforcement", "The reinforcement object to extract the count from.")]
+        [Description("Return the number of reinforcing bars in one a LongtudinalReinforcement object.")]
+        [Input("reinforcement", "The LongitudinalReinforcement object to extract the count from.")]
         [Output("count", "Number of bars in the provided LongitudinalReinforcement.")]
         public static int ReinforcingBarCount(this LongitudinalReinforcement reinforcement)
         {
-            return LayoutCount(reinforcement.RebarLayout as dynamic);
+            return reinforcement.IsNull() ? null : LayoutCount(reinforcement.RebarLayout as dynamic);
```

Comment:
Not sure why this is not flagged by the compiler tbh, but need to change to this I think:

```suggestion
            return reinforcement.IsNull() ? 0 : LayoutCount(reinforcement.RebarLayout as dynamic);
```